Datatable/Data/DatatableQuery.php

```diff
@@ -778,9 +778,21 @@ public function getResponse($buildQuery = true)
         $fullOutput = array_merge($outputHeader, $output);
         $fullOutput = $this->applyResponseCallbacks($fullOutput);
 
-        $json = $this->serializer->serialize($fullOutput, 'json');
-        $response = new Response($json);
-        $response->headers->set('Content-Type', 'application/json');
+        switch ($outputType){
+            case "array":
+                if($fullOutput['recordsTotal']>0){
+                    $response=$fullOutput;
+                }else{
+                    throw new Exception('No existen registros suficientes para generar el reporte.');
```

Comment:
`No existen registros suficientes para generar el reporte.` should be an string with english translation